### PR TITLE
Updates per IETF guidelines

### PR DESCRIPTION
**Pull request by bfussell**
There are still sections that require work and we will address those individually. Those are:
introduction, references, terminology, traceability, error messages, IANA considerations and contributors.
Some may simply be removed.

### CODE REVIEW

File: src/draft-fussell-acvp-spec-00.xml [hunk 1]
```diff
@@ -145,25 +145,35 @@
...
-        <t>The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD
-          NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
-          described in <xref target="RFC2119">RFC 2119</xref>.</t>
-      </section>
+      <t>The current methods used for cryptographic module testing are limited
+         by manual test vector exchange, long validation cycles and cost. 
+         These problem areas impact development cycles, hinder adoption of new
+         technologies and contribute to end customer anxiety when deciding whether
+         to deploy unvalidated new crypto with CVE fixes over previously deployed and
+         validated crypto. In order to compat these issues validation authorities
```
Comment (elliemdaw) [inline src/draft-fussell-acvp-spec-00.xml]: "combat" ? :)

File: src/draft-fussell-acvp-spec-00.xml [hunk 2]
```diff
@@ -145,25 +145,35 @@
...
+         need a more automateable method of cryptographic testing.  The ACV Protocol(ACVP)
```
Comment (celic) [inline src/draft-fussell-acvp-spec-00.xml]: automatic sounds a bit more natural

File: src/draft-fussell-acvp-spec-00.xml [hunk 3]
```diff
@@ -337,13 +348,13 @@
...
-          +-------------+------------------+-------------------------+
-             server          path prefix          resource
-          +-------------+------------------+-------------------------+
-  https://acvts.nist.gov/validation/acvp/v1/testSession/1/vectorSet/1
-          +-------------+----------+----+--+-------------------------+
-                           context  API   |
-                                       version
+         +-------------+--------------------+---------------+
+            server          path prefix          resource
+         +-------------+------==------------+---------------+
```
Comment (celic) [inline src/draft-fussell-acvp-spec-00.xml]: == should be --

File: src/draft-fussell-acvp-spec-00.xml [hunk 4]
```diff
@@ -337,13 +348,13 @@
...
+ https://acvts.nist.gov/acvp/validation/acvp/login
+         +-------------+----------+----+--+-----------------+
```
Comment (celic) [inline src/draft-fussell-acvp-spec-00.xml]: lots of extra + for partitions that were removed. Should match top bounding bar.

File: src/draft-fussell-acvp-spec-00.xml (hunk 1 above)
Comment (bfussell) [reply src/draft-fussell-acvp-spec-00.xml]: thanks !

File: src/draft-fussell-acvp-spec-00.xml (hunk 2 above)
Comment (bfussell) [reply src/draft-fussell-acvp-spec-00.xml]: probably not  a real word anyway ;)

File: src/draft-fussell-acvp-spec-00.xml (hunk 3 above)
Comment (bfussell) [reply src/draft-fussell-acvp-spec-00.xml]: thanks !

File: src/draft-fussell-acvp-spec-00.xml (hunk 4 above)
Comment (bfussell) [reply src/draft-fussell-acvp-spec-00.xml]: got it.

File: artifacts/draft-fussell-acvp-spec-00.html [hunk 5]
```diff
@@ -373,105 +373,107 @@
...
   <link rel="schema.dct" href="http://purl.org/dc/terms/" />
 
   <meta name="dct.creator" content="Fussell, B., Ed., Vassilev, A., Ed., and H. Booth" />
   <meta name="dct.identifier" content="urn:ietf:id:draft-fussell-acvp-spec-00" />
-  <meta name="dct.issued" scheme="ISO8601" content="2018-06" />
-  <meta name="dct.abstract" content="This document defines the Automated Cryptographic Validation Protocol(ACVP)." />
-  <meta name="description" content="This document defines the Automated Cryptographic Validation Protocol(ACVP)." />
+  <meta name="dct.issued" scheme="ISO8601" content="2018-8-24" />
+  <meta name="dct.abstract" content="The ACV Protocol provides a method for communication between a cryptographic module that is embedded inside of a device or otherwise running on a platform accessible via computer network, and an external testing system, using standard network communication interfaces and protocols. This communication protocol can also be used to validate the correctness of the implementations of the algorithms in the cryptographic module with a validation authority." />
```
Comment (fliphil) [inline artifacts/draft-fussell-acvp-spec-00.html]: "correctness of the algorithm implementations" sounds a bit more natural?

File: src/draft-fussell-acvp-spec-00.xml [hunk 6]
```diff
@@ -354,13 +365,14 @@
       </section>
 
       <section title="HTTP URI Resources">
-        <t>In the table below, any part of a resource path enclosed in curly braces, { or }, are
-          replaced by an instance of what is described in the brackets. For example {testSessionId}
-          could be replaced with 1.</t>
+        <t>In the table below, any part of a resource path enclosed in curly braces, { or },
```
[the remark this answers is not in the document]
Comment (bfussell) [reply src/draft-fussell-acvp-spec-00.xml]: k

File: artifacts/draft-fussell-acvp-spec-00.html (hunk 5 above)
Comment (bfussell) [reply artifacts/draft-fussell-acvp-spec-00.html]: k

**Pull request merged**